cfgov/wellbeing/jinja2/wellbeing/about.html

```diff
@@ -23,132 +23,83 @@
 {% block content_main %}
     <h1>Why financial well-being?</h1>
     <p class="lead-paragraph">
-        We developed the Financial Well-Being Scale to help answer
-        an important question: Am I better off than before?
+        At the CFPB, we work to help consumers like you take control
+        of your financial life to reach you own life goals,
+        achieve financial peace of mind, and avoid pitfalls that can derail you.
+    </p>
+    <p>
+        The ultimate goal of our work is to help you
+        improve your financial well-being.
+        Financial well-being means how much your financial situation
+        and money choices provide you with security and freedom of choice.
+
+    </p>
+    <p>
+        We developed a questionnaire and a scoring method as a tool
+        that can help you take stock of your financial well-being.
     </p>
     <p>
-        When people take action to improve their financial lives—and
-        when they get help from others, maybe through a
-        friend, coach, book, or financial services professional—one of the
-        key questions they ask themselves afterward is,
+        Maybe you’re planning to take action to improve your financial
+        life—on your own or with help from others,
+        maybe through a friend, coach, book, or financial services professional.
+        One of the key questions you might ask yourself afterward is,
         “Am I better off than before?”
     </p>
     <p>
-        To answer that question, you can
-        <a href="../">use the CFPB’s financial well-being questionnaire</a>
-        and your resulting score.
-        Together, they let you accurately and consistently quantify,
-        and therefore observe, something that is not directly observable—the
-        extent to which your financial situation and the financial capability
-        that you have developed provide you with security and freedom of choice.
+        To answer that question, you can use our financial well-being
+        questionnaire and your resulting score.
+        Together, they let you accurately and consistently measure
+        your financial well-being before and after
+        taking steps to improve your financial situation.
     </p>
 
-    <h2>What is financial well-being?</h2>
+    <h2>Why doesn’t this questionnaire ask about financial details?</h2>
     <p>
         Because each person’s situation is unique and subjective,
         it is hard to describe financial well-being using only numbers
         like income, net worth, or credit score.
     </p>
     <p>
-        Instead, we asked people around the country
+        Instead,
+        <a href="/about-us/blog/four-elements-define-personal-financial-well-being/">
```

Comment:
Is it possible to dynamically look up Wagtail content from a Django child app template?

Reply:
I may have been too optimistic about this -- I was thinking we could use something like what the Wagtail RTE uses for internal page links, like a `reverse` with the page slug. This might be possible with [the `slugurl` templatetag](http://docs.wagtail.io/en/v1.9/topics/writing_templates.html#slugurl) but I'm not sure we can easily access that from Jinja templates.